docker/postgres-setup/Dockerfile

```diff
@@ -6,6 +6,6 @@ COPY docker/postgres-setup/init.sql /init.sql
 COPY docker/postgres-setup/init.sh /init.sh
 RUN chmod 755 init.sh
 
-ENV DATAHUB_DB_NAME="datahub"
+ENV DATAHUB_DB_NAME=$POSTGRES_DATABASENAME
```

Comment:
Why do we need a separate env variable?

Reply:
Can we just use DATAHUB_DB_NAME ?

Reply:
I guess we don't need it. Initial intention is for the users to be able set their own database name through the env. But if it's unnecessary, we can remove it!

Reply:
So DATAHUB_DB_NAME already does that. Here it seems like it's always set to "datahub" but it does use the appropriate db name during setup.(This is the case for mysql-setup, where I tested with different db names)

Reply:
true...dbeec80